### ISSUE TITLE
Setting Div hard to see

### ISSUE THREAD

**Issue by biofects** [Nihvel/your_name#12]
Hello when I am using this great theme the settings pop-up div is transparent and the text is hard to read. Is there a way to make the pop-up div a solid color?

**Comment by Nihvel** [comment]
Hi! Yes, unfortunately I know what you mean. It's hard for me to read in there too. I created this theme for the dashboard frontend only and fixed small issues for the "backend" (Settings, developer tools...) pages.

The transparency affects multiple areas in HA, as you noticed. I never looked into finding a fix for it. When developing the theme I recall having lots of difficulties because of it. But I did not want to renounce. So I settled.

I personally use Your Name. for the mobile view and tablets around home. My configuration is mostly yaml files because I'm unable to use the frontend for creating automation / scripts and such, where with yaml I feel home.

The times I use the frontend is when I have to use the Settings (Backup, Logs) and Developer tools (States, Services [...]). In those cases I login from the laptop, which uses the default theme.
HA gives you the possibility to use different themes for the same user, though I don't know if this is a feature or a bug. 

If I will ever find a fix for it I will make sure to upload it.

edit: I'm happy however that you like this theme!
